docs/enterprise-guide/saml-google.md

```diff
@@ -6,23 +6,54 @@ title: Setting up SAML with Google
-| ----------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| URL the IdP should redirect back to | Google Admin console > Service Provider Details > ACS URL. This is the same as your Metabase URL -- it should start with `https://` and end with `/auth/sso`. |
-| Users email attribute               | Google Admin console > Attribute mapping                                                                                                                      |
-| Users first name attribute          | Google Admin console > Attribute mapping                                                                                                                      |
-| Users last name attribute           | Google Admin console > Attribute mapping                                                                                                                      |
-| SAML Identity Provider URL          | Google Admin console > Google Identity Provider details > Copy the **SSO URL**                                                                                |
-| SAML Identity Provider Certificate  | Google Admin console > Google Identity Provider details > Download certificate                                                                                |
-| SAML Application Name               | Google Admin console > Google Identity Provider details > Copy the **Entity ID**                                                                              |
-| SAML Identity Provider Issuer       | Google Admin console > Google Identity Provider details > Download metadata                                                                                   |
+1. Set up a [custom SAML app](https://support.google.com/a/answer/6087519) in your [Google admin console](admin.google.com).
+2. As you follow Google's instructions, you'll need to:
+  - [Save information about Google for Metabase](#saving-google-idp-information-for-metabase).
+  - [Provide Google info to Metabase](#filling-out-the-metabase-saml-form).
+  - [Provide Metabase info to Google ](#filling-out-service-provider-details).
+  - [Set up attribute mappings in Google](#setting-up-attribute-mappings).
+
+See [authenticating with SAML](../enterprise-guide/authenticating-with-saml.html) for general SAML info.
+
+## Saving Google IdP info for Metabase
+
+On the **Google Identity Provider details** page:
+
+1. Download the **IdP metadata**.
+2. Copy the **SSO URL**.
+3. Download the **certificate**.
+
+## Filling out the Metabase SAML form
+
+1. From your Google **IdP metadata**, locate the **issuer**.
+   - The **issuer** should be a URL that starts with `https://accounts.google.com/o/saml2/`.
```

Comment:
@flamber is this correct? I took this from your screenshot, but not sure if it generally applies.

Reply:
I've only seen this style, so I think it's general, but I don't know.